src/util/Injected.js

```diff
@@ -185,7 +185,7 @@ exports.LoadUtils = () => {
         const chat = window.Store.Chat.get(chatId);
         if (!chat) return null;
 
-        return window.WWebJS.getChatModel(chat);
+        return await window.WWebJS.getChatModel(chat);
```

Comment:
why `return await` here?

Reply:
some insight: https://eslint.org/docs/rules/no-return-await

Reply:
Yep, you're right. Adding `async` alone was enough here. This was changed on another commit- I'm just merging changes back, so I'm resolving this here. I'll do some cleanup soon when I do a slight refactor I'm planning of a couple things.